packages/core/admin/package.json

```diff
@@ -114,6 +113,7 @@
     "react-fast-compare": "^3.2.0",
     "react-helmet": "^6.1.0",
     "react-intl": "5.20.2",
+    "react-is": "^17.0.2",
```

Comment:
I'm still wondering where this comes from. We don't use the package explictly anywhere ...